ElrondNetwork/elrond-go
# En 7316 add call value sc query

Added two optional parameters to API endpoint `vm-values` (`caller` and `value`) and extended the functionality in order to be able to include the the `callerAddr` and `callValue` inside the SC query service.

Testing steps:
1. start a testnet
2. make a POST request to API endpoint `/vm-values/int` for example in which you will specify the caller address and the call value. Can be a `stake` transaction.
3. you should see the vm output in the response
